DEVELOPING.rst

```diff
@@ -1,3 +1,43 @@
+Local development
+*****************
+
+Nomenclature uses poetry for local development. Follow these steps to get setup:
+
+
+```bash
+# clone the nomenclature repository
+git clone git@github.com:IAMconsortium/nomenclature.git
+cd nomenclature
+
+# Install Poetry, minimum version >=1.2 required
+curl -sSL https://install.python-poetry.org | python -
+
+# You may have to reinitialize your shell at this point.
+source ~/.bashrc
+
+# Activate in-project virtualenvs
+poetry config virtualenvs.in-project true
+
+# Add dynamic versioning plugin
+poetry self add "poetry-dynamic-versioning[plugin]"
+
+# Install dependencies
+# (using "--with dev,docs,server" if dev and docs dependencies should be installed as well)
+poetry install --with dev,docs
+
+# Activate virtual environment
+poetry shell
+
+# Copy the template environment configuration
+cp template.env .env
+
+# Add a test platform
```

Comment:
I guess that this is not relevant for nomenclature

Reply:
Ah ups, good catch